01-js-intro-and-jquery/js/javascript.js

```diff
@@ -0,0 +1,84 @@
+
+
+var functionFadeToggleSection = (function() {
+  //Private variables
+  var selectedSection = $('section.hidden-section');
```

Comment:
Just a heads up!
As a general practice, try to use more generic names for these kind of variables. For example, this could be ``hiddenContainer``, since you could use any element and not just a ``<section />``.
``text`` variable for example is a case where you're being too generic maybe (since it could refer to any kind of text in any part of the page). Try to think about the purpose this element will accomplish (``nameInput``, for example).